fullscreen-api/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Fullscreen API Example</title>
+  <script src="main.js"></script>
+  <link href="main.css" rel="stylesheet">
+</head>
+
+<body>
+  <div id="contents">
+    <p>A quick example for the use of the <code>Fullscreen</code> API. Press <code>ENTER</code> key to toggle Full screen.</p>
```

Comment:
Rephrase to "A quick example to demonstrate usage of the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API">Fullscreen API</a> API. Press <kbd>Enter</kbd> to toggle fullscreen mode."